cmake/nebula/TimeTrace.cmake

```diff
@@ -0,0 +1,6 @@
+if(CMAKE_CXX_COMPILER_ID MATCHES ".*Clang")
+  option(ENABLE_BUILD_WITH_TIME_TRACE "Enable -ftime-trace to generate time tracing .json files on clang" OFF)
```

Comment:
It is off by default？

Reply:
yes, it's for developer to trace compile time, not intend to be used for user.

Reply:
reasonable